packages/assets-controllers/jest.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = merge(baseConfig, {
   // An object that configures minimum threshold enforcement for coverage results
   coverageThreshold: {
     global: {
-      branches: 91.07,
+      branches: 90,
```

Comment:
We should revert that now that we've added other tests:

```suggestion
      branches: 91.07,
```

Reply:
Outdated! has been done :D